src/collections/_documentation/workflow/search.md

```diff
@@ -218,12 +217,22 @@ Below is a list of Event-level tokens reserved and known to Sentry:
 
 : Restrict results to events with a matching stack property.
 
-&nbsp;
 ### Custom Tags
 
-Additionally, you can use any tag you’ve specified as a token.
+Additionally, you can use any tag you’ve specified as a token. Tags are various key/value pairs that get assigned to an event, and you can use them later as a breakdown or quick access to finding related events.
+
+Most SDKs generally support configuring tags by configuring the scope:
+
+{% include components/platform_content.html content_dir='set-tag' %}
+
+Several common uses for tags include:
+
+-   The hostname of the server
+-   The version of your platform (for example, iOS 5.0)
+-   The user’s language
+
+For more information, see [full documentation on Tagging Events]({%- link _documentation/enriching-error-data/context.md -%} #tagging-events).
```

Comment:
```suggestion
For more information, see [full documentation on Tagging Events]({%- link _documentation/enriching-error-data/context.md -%}#tagging-events).
```

While not strictly necessary because the `-` in `-%}` trims the trailing whitespace, I'd prefer we don't include the space before the hash.